src/constants/supportedNetworks.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ChainParams,
+  NetworkChains,
+  SupportedNetworks,
+} from "../types/supportedNetworks";
+
+export const SUPPORTED_NETWORKS: SupportedNetworks = {
+  Ethereum: {
+    icon: "images/ethereum-logo.png",
+    chainId: 1,
+  },
+  Avalanche: {
+    icon: "images/avalanche-logo.png",
+    chainId: 43114,
+  },
+  "BNB Chain": {
+    icon: "images/bnb-logo.png",
+    chainId: 56,
+  },
+  Polygon: {
+    icon: "images/matic-logo.png",
+    chainId: 137,
+  },
+};
+
+export const NETWORK_CHAINS: NetworkChains = {
```

Comment:
I don't think `NETWORK_CHAINS` is necessary because that info is also in `CHAIN_PARAMS`.

Reply:
The reason I made the `NETWORK_CHAINS` object is because there is no way to access the chain name (like "Ethereum" or "Avalanche") only with a chainId. We need the network name to be displayed on the button. `nativeCurrency` has a lot of data, but not the actual network names. Should we add network names to the nativeCurrency object?

Reply:
See comment above: use `chainNames`.

Reply:
(same comment as above)
I consolidated all of the objects I created into 1 huge object called CHAIN_PARAMS. This references @airswap/constants where possible